packages/wonder-blocks-banner/src/components/__docs__/banner.stories.tsx

```diff
@@ -12,6 +11,7 @@ import {LabelSmall} from "@khanacademy/wonder-blocks-typography";
 
 // @ts-expect-error [FEI-5019] - TS2305 - Module '"@storybook/react"' has no exported member 'StoryComponentType'.
 import type {StoryComponentType} from "@storybook/react";
+import Banner from "@khanacademy/wonder-blocks-banner";
```

Comment:
Why the move?

Reply:
Bad autofixing from `import/order`.  I can fix this.